document/js-api/index.bs

```diff
@@ -1336,20 +1355,22 @@ The <dfn attribute for="Exception">stack</dfn> getter steps are:
 
 <h4 id="js-exceptions">JavaScript exceptions</h4>
 
-The <dfn>JavaScript exception tag</dfn> is a [=tag address=] reserved by this
-specification to distinguish exceptions originating from JavaScript.
+The <dfn>JavaScript exception tag</dfn> is a [=tag address=] associated with
+the surrounding agent. It is allocated in the agent's [=associated store=] on
+first use and cached. It always has the [=tag type=] « [=externref=] » → « ».
 
-For any [=associated store=] |store|, the result of
-[=tag_parameters=](|store|, [=JavaScript exception tag=]) must be « ».
 
 <div algorithm>
 
-To <dfn for=WebAssembly>throw</dfn> with a [=tag address=] |type|, a matching [=list=] of WebAssembly values |payload|, and an [=externref=] |opaqueData|, perform the following steps:
-
-1. Unwind the stack until reaching the *catching try block* given |type|.
-1. Invoke the catch block with |payload| and |opaqueData|.
+To <dfn>get the JavaScript exception tag</dfn>, perform the following steps:
 
-Note: This algorithm is expected to be moved into the core specification.
+    1. If the [=surrounding agent=]'s associated [=JavaScript exception tag=] has been initialized,
+        1. return the [=surrounding agent=]'s associated [=JavaScript exception tag=]
+    1. Let |store| be the [=surrounding agent=]'s [=associated store=].
+    1. Let (|store|, |tagAddress|) be [=tag_alloc=](|store|, « [=externref=] » → « »).
+    1. Set the current agent's [=associated store=] to |store|.
+    1. Set the current agent's associated [=JavaScript exception tag=] to |tagAddress|.
```

Comment:
Does this mean tag == tagAddress? Somewhere above in this doc contains
```
Set map[tagAddress] to tag.
```
So they seem to be different.

And this PR doesn't include how Wasm should handle the JS exception tag, right?

Reply:
Line 1208 is where you're referring to. There `tag` and `tagAddress` are both basically function parameters, and `tag` is a JS object and `tagAddress` is an address as defined in the wasm runtime spec. Here `tagAddress` is basically a local variable (a wasm address), a return value of `tag_alloc`; and `tag` isn't a variable at all. This function just returns the address.

And yes,  #269 is the PR that allows actually exposing the JS exception tag (which by extension allows it to be imported into a wasm module).